runtime/vm/ContinuationHelpers.cpp

```diff
@@ -265,6 +265,11 @@ copyFieldsFromContinuation(J9VMThread *currentThread, J9VMThread *vmThread, J9VM
 	els->jitGlobalStorageBase = (UDATA*)&continuation->jitGPRs;
 	els->i2jState = continuation->i2jState;
 	vmThread->entryLocalStorage = els;
+
+	/* Disable the JIT artifact cache on the walk thread.  It provides little performance
+	 * benefit to a single walk and the cache memory must be managed.
+	 */
+	vmThread->jitArtifactSearchCache = (void*)((UDATA)vmThread->jitArtifactSearchCache | J9_STACKWALK_NO_JIT_CACHE);
```

Comment:
When we first set this field, we use a compareAndSwap:
https://github.com/eclipse-openj9/openj9/blob/cf3c1ee0d3b38d3f7af41e1e51e4c5f7ca23f862/runtime/codert_vm/jswalk.c#L1510


Therefore, when setting this to NULL, shouldn't we also do the same?

Reply:
This API will only be used to populate stack allocated thread with data from continuation. the thread reference isn't shared so there's no need for CAS.

Also the CAS in jswalk.c is to avoid duplicate allocation which wont happen here since nothing is being allocated.